pyramid/config/views.py

```diff
@@ -2102,3 +2100,86 @@ def register():
         intr['spec'] = spec
 
         config.action(None, callable=register, introspectables=(intr,))
+
+    def add_cache_buster(self, config, spec, cachebust, explicit=False):
+        if config.registry.settings.get('pyramid.prevent_cachebust'):
+            return
```

Comment:
Should we move this check down into register() so that if you do config.add_cache_buster(), config.registry.settings['pyramid.prevent_cachebust'] = True it works the same way? I can imagine scenarios where a user may want to disable cache busting but may have already called config.include() or something along those lines.